Sources/UI/Chat/Chat View Controller/ChatViewController.swift

```diff
@@ -308,6 +308,19 @@ open class ChatViewController: ViewController, UITableViewDataSource, UITableVie
         defaultActionsContextMenu(from: cell, for: message, locationInView: locationInView)
     }
     
+    /// Creates a chat view controller for the message being replied to.
```

Comment:
Can you add a note about how one should override this to add custom styling to their threads?